### PR TITLE
Add yargs, CLI help section, specify port with -p

### PR DESCRIPTION
**Pull request by samohovets**
- [x] Add yargs
- [x] Add CLI help section
- [x] Specify port with -p or --port

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -25,6 +25,26 @@ const setup = async () => {
 };
 exports.setup = setup;
 
+// CLI config
```
Comment (yamalight) [inline src/index.js]: This shouldn't be in the `src/index.js` - it's a part of CLI.

Comment (yamalight) [reply src/index.js]: I'd move all of that into `bin/index.js` and use simple yargs setup

**Review by yamalight — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (samohovets) [reply src/index.js]: Hm, okay 🤔

File: bin/particula.js [hunk 2]
```diff
@@ -1,13 +1,30 @@
 #!/usr/bin/env node
-const yargs = require('yargs');
+const yargs = require("yargs");
```
Comment (yamalight) [inline bin/particula.js]: Please revert to single quotes

File: bin/particula.js [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
+      }
+    },
+    handler({ port }) {
+      if (isNaN(port)) {
+        console.error("Invalid port");
+        process.exit(1);
+      }
+      start(port);
+    }
+  }).argv;
```
Comment (yamalight) [inline bin/particula.js]: Please add line-break at the end of file

**Review by yamalight — CHANGES_REQUESTED** [review]

**Comment by yamalight** [comment]
Looks good, thanks!

**Pull request merged**